main_test.go

```diff
@@ -0,0 +1,75 @@
+package meilisearch
+
+import (
+	"os"
+	"testing"
+)
+
+type docTest struct {
+	ID   string `json:"id"`
+	Name string `json:"name"`
+}
+
+type docTestBooks struct {
+	BookID int    `json:"book_id"`
+	Title  string `json:"title"`
+	Tag    string `json:"tag"`
+}
+
+func deleteAllIndexes(client *Client) (ok bool, err error) {
+	list, err := client.Indexes().List()
+
+	if err != nil {
+		return false, err
+	}
+
+	for _, index := range list {
+		client.Indexes().Delete(index.UID)
+	}
+
+	return true, nil
+}
+
+func TestMain(m *testing.M) {
+	var client = NewClient(Config{
+		Host: "http://localhost:7700",
+	})
+	deleteAllIndexes(client)
+	m.Run()
+	deleteAllIndexes(client)
+	os.Exit(0)
+}
+
+func TestClientIndexes_deleteAllIndexes(t *testing.T) {
```

Comment:
Is it a test for `deleteAllIndexes` function?

Reply:
This used to be in a different file, name is wrong, will fix now!